apps/researcher/src/lib/community/definitions.ts

```diff
@@ -0,0 +1,28 @@
+export interface Community {
+  id: string;
+  name: string;
+  slug: string;
+  description?: string;
+  attributionId?: string;
+  imageUrl: string;
+  createdAt: number;
+  membershipCount?: number;
+  licence?: string;
```

Comment:
Question for @sdevalk: I chose the term `licence`. (British English) I see the enricher API uses `license` (American English). Maybe we should choose one, are we using British or American English?

Reply:
For the data model of the 'backend of the frontend' I use elements from the Schema.org vocabulary. This vocab uses `en-us`, e.g. [license](https://schema.org/license). I prefer to use this style because most models and vocabs that I know of use it. However, for the display to the user we can use `en-gb`.

Reply:
I have renamed it to `license`. I see Gertjan also uses `license` in the text he provided. I will keep it the way Gertjan wrote it.